src/tests/FunctionalTests/iOS/Simulator/CoreCLR.Interpreter/Program.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+public static class Program
+{
+    [DllImport("__Internal")]
+    public static extern void mono_ios_set_summary(string value);
```

Comment:
feel free to rename this across the board so we don't have confusing mono names in coreclr apps (doesn't need to be in this PR)